invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/table.overrides

```diff
@@ -3,68 +3,94 @@
 ***********************************************/
 
 .ui.table {
-  @media all and (max-width: @largestTabletScreen) {
-    &:not(.unstackable) tbody tr {
-      padding-top: 0;
-    }
 
-    &:not(.unstackable) tbody, tr, tr > th, tr > td {
-      width: auto !important;
-      display: block !important;
-    }
+  &.files {
+    border: 0;
 
-    &:not(.unstackable) tr > th {
-        display: none !important; // Responsive: header is hidden and data-label is aligned with the table data.
+    th.hidden {
+      color: transparent;
+      user-select: none;
     }
+  }
 
-    &:not(.unstackable) tr > td.responsive-main-cell {
-      background-color: lighten(@borderColor, 65%);
-      margin-bottom: 1rem;
-      padding: 1rem !important; //overriding !important on .ui.table:not(.unstackable) tr > td
-    }
+  td.file-upload-pending {
+    text-align: center;
+  }
 
-    td[data-label]  {
-      margin-top: .75rem;
+  .ui.blue.medium.file-upload-progress {
+    &.progress,
+    &.success {
+      &.active,
+      &.disabled {
+        margin-bottom: 0;
 
-      &:before {
-        content: attr(data-label);
-        font-weight: bold;
-        margin-right: 1rem;
-        margin-left: 2rem;
+        &.error {
+          margin-bottom: 0;
+        }
       }
     }
   }
 
-  tbody tr td {
-    &.mobile-table-cell {
-      @media all and (max-width: @largestTabletScreen) {
-        display: inline-grid !important;
-        grid-template-columns: min-content auto;
-        grid-column-gap: 0.5em;
-        align-items: center;
-        width: 50% !important;
-        border-top: 0px;
-      }
-      @media all and (max-width: @largestMobileScreen) {
-        width: 100% !important;
+  @media all and (max-width: @largestTabletScreen) {
+    &:not(.unstackable) {
+
+      thead {
+        display: none !important; // Responsive: header is hidden and data-label is aligned with the table data.
       }
-    }
-  }
-}
 
-.ui.table.files {
-  border: 0;
+      tbody {
 
-  th.hidden {
-    color: transparent;
-    user-select: none;
-  }
+        tr {
+          padding-top: 0;
+          padding-bottom: 0;
+          box-shadow: none !important; // overriding !important on .ui.table:not(.unstackable) tr
 
-  @media all and (max-width: @largestTabletScreen) {
-    td {
-      text-align: left
+          &:not(:first-child) {
+            border-top: 1px solid @borderColor;
+          }
+
+          td {
+            border-top: 0px;
+            width: 100% !important;
+            padding: 1rem !important; // overriding !important on .ui.table:not(.unstackable) tr > td
+
+            &:nth-child(2n+1) {
+              background-color: lighten(@borderColor, 75%);
+            }
+
+            &:not(.right.aligned) {
+              text-align: left;
+            }
+
+            &[data-label]  {
+              display: inline-grid !important;
+              grid-template-columns: [col1-start] 30% [col2-start] 70% [end];
```

Comment:
Why did you need to define the columns here instead of using the Grid from semantic-ui?

Reply:
Because the `td[data-label]` only has a `::before`, it's not an actual element, so it's not controllable like that. Also, this styling is to make everything work for `.ui.table`s out of the box so that there is no need to customize the markup to make it responsive (other than adding a `data-label`-attribute to `td`s that should have a label for the mobile version :)